config/tech-docs.yml

```diff
@@ -1,38 +1,38 @@
 # Host to use for canonical URL generation (without trailing slash)
-host: 
+host:
 
 # Header-related options
 show_govuk_logo: true
-service_name: My First Service
-service_link: https://www.larry-the-cat.service.gov.uk
+service_name: Accounts
+service_link: https://www.gov.uk/
```

Comment:
Should this be https://www.account.publishing.service.gov.uk/ ?

Reply:
Normally it would be the product page for the service. Other tech docs have this, for example https://github.com/alphagov/pay-tech-docs/blob/master/config/tech-docs.yml. 

We don't have a product page for Accounts yet I don't think?

Reply:
No, we don't